src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/00 About scenarios/02 VU allocation.md

```diff
@@ -0,0 +1,102 @@
+      preAllocatedVUs: 4,
+      rate: 8,
+      timeUnit: "1m",
+    },
+  },
+};
+```
+
+
+<Blockquote mod="attention" title="">
+
+In cloud tests, **both `preAllocatedVUs` and `maxVUs` count against your subscription.**
+
+When planning a test, consider doing a trial initialization on a local machine to ensure you're allocating VUs efficiently.
+
+</Blockquote>
+
+## How k6 uses allocated VUs
+
+Before an arrival-rate scenario starts, k6 first initializes the number of `preAllocatedVUs`.
+When the test runs,
+the number of available `preAllocatedVUs` determines how many iterations k6 can start.
+k6 tries to reach the target iterations per second,
+and one of two things can happen:
+
+| If the executor       | Then..                                                                                                                                 |
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| Has enough VUs        | the extra VUs are "idle," ready to be used when needed.                                                                                |
+| Has insufficient VUs. | k6 emits a [`dropped_iterations` metric](/using-k6/scenarios/about-scenarios/dropped-iterations) for each iteration that it can't run. |
+
+## Iteration duration affects the necessary allocation
+
+The necessary allocation depends on the iteration duration:
+Longer durations need more VUs.
+
+In a perfect world, you could estimate the number of pre-allocated VUs with this formula:
+
+```
+preAllocatedVUs = [median_iteration_duration * rate] + constant_for_variance
+```
+
+In the real world, if you know _exactly_ how long an iteration takes, you likely don't need to run a test.
+What's more, as the test goes on, iteration duration likely increases.
+If response times slow so much that k6 lacks the VUs to start iterations at the desired rate,
+the allocation might be insufficient and k6 will drop iterations.
+
+To determine your strategy, you can run tests locally and gradually add more pre-allocated VUs.
+As dropped iterations can also indicate that the system performance is degrading, this early experimentation can provide useful data on its own.
+
+## You probably don't need `maxVUs`
+
+The arrival-rate executors also have a `maxVUs` property.
+If you set it, k6 runs in this sequence:
+1. Pre-allocate the `preAllocatedVUs`.
+1. Run the test, trying to reach the target iteration rate.
+1. If the target exceeds the available VUs, allocate another VU.
+1. If the target still exceeds available VUs, continue allocating VUs until reaching the number set by `maxVUs`.
+
+Though it seems convenient, you should avoid using `maxVUs` in most cases.
+Allocating VUs has CPU and memory costs, and allocating VUs as the test runs **can overload the load generator and skew results**.
+If you're running in k6 Cloud, the `maxVUs` will count against your subscription.
```

Comment:
```suggestion
If you're running in k6 Cloud, the `maxVUs` will count against your subscription.
This is because k6 must allocate sufficient resources for `maxVUs` to be initialized, even if they never are.
```